docs/getting-started/self-healing-rollback.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[self-healing-rollback]]
+= Configure self-healing rollback for Windows endpoints
+
+{endpoint-cloud-sec}'s self-healing feature rolls back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features. All activity on the host reverts to its state five minutes before the prevention alert.
```

Comment:
I think "Endpoint and Cloud Security feature" is appropriate here, because it's part of that integration and this sentence is talking about what _the feature_ is and does, not how it's configured. After the rename of the integration, there isn't really anything called "Endpoint Security" anymore. @ferullo @joe-desimone thoughts?

Reply:
I think Endpoint and Cloud Security is appropriate here.